src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -130,6 +188,11 @@
                     }
                 }
             }
+        },
+        watch: {
```

Comment:
Nice... didn't know this was possible. If I had then `changedCommit` would have been done here.